src/transforms/transform-code-scanning-alert.test.ts

```diff
@@ -182,41 +181,40 @@ describe("code_scanning_alert transform", () => {
 				gitHubInstallationId,
 				jiraHost
 			);
-			expect(result?.vulnerabilities[0]).toMatchObject(
-				{
-					"additionalInfo": {
-						"content": "CodeQL"
+			expect(result?.vulnerabilities[0]).toMatchObject({
```

Comment:
/nit
What's changed in this file? Can you make it dosen't shown that much change here?

Reply:
The only change is the removal of the double quotes for the keys within the object.